src/standalone/Vegetation/canopy_parameterizations.jl

```diff
@@ -567,7 +601,7 @@ the total net carbon assimilation (`An`), the extinction coefficient (`K`),
 leaf area index (`LAI`) and the clumping index (`Ω`).
 """
 function compute_GPP(An::FT, K::FT, LAI::FT, Ω::FT) where {FT}
-    GPP = An * (1 - exp(-K * LAI * Ω)) / K
+    GPP = An * (1 - exp(-K * LAI * Ω)) / (K * Ω)
```

Comment:
@braghiere since beta is now applied to gs directly, and not to An, is GPP affected by beta at all?

just wanted to double check this - i remember we had briefly mentioned this earlier.

Reply:
That is a good point, but no. That's why, in my point of view, it would be better to apply beta on An like Eq (15) in: 
Clark, D. B., et al. 2011: The Joint UK Land Environment Simulator (JULES), model description – Part 2: Carbon fluxes and vegetation dynamics, Geosci. Model Dev., 4, 701–722, https://doi.org/10.5194/gmd-4-701-2011

I think right now GPP is indirectly being impacted by gs only.

Reply:
RMSE of GPP is lower when applying beta to An, as soon as we remove it we lose the decrease of GPP during the summer. My question is, if beta is applied back to An as in equation 15, should it then not be applied in the dark respiration function and match equation 13 from that paper? If it is applied to both as it was originally, then it gets applied to dark respiration twice

Reply:
thanks, Edward! I think we are getting at the same thing...

How is GPP affected by gs? We use `gs` to compute the canopy conductance here: https://github.com/CliMA/ClimaLSM.jl/blob/3b2c8bcaadce3d439bba14fd1ab63ff119146550/src/standalone/Vegetation/canopy_parameterizations.jl#L627 and that is used when we compute transpiration. maybe I am missing how `gs` affects GPP? our formula for GPP has An, K, omega, and LAI, none of which depend on gs, right?

Regarding applying it to net photosynthesis: I think we had it OK before?
https://github.com/CliMA/ClimaLSM.jl/blob/ae8985563721f5d44d35b3907e08c077f9c5461d/src/standalone/Vegetation/canopy_parameterizations.jl#L506
but Rd already had the beta. so whatever comes out has just one factor of beta applied to all terms...
but perhaps it would be clearer to remove from Rd formula, and just apply it in net_photosynthesis?

Reply:
Either way, this is not necessary for this PR. If we want to table this for later discussion (perhaps see the impact of using different functions for beta instead of removing it from An as we thought about a couple weeks ago?) I can remove it from this PR and let this one just focus on the diffuse radiation function.